modules/rgbd/src/hash_tsdf.hpp

```diff
@@ -31,6 +31,7 @@ class HashTSDFVolume : public Volume
     float truncDist;
     float truncateThreshold;
     int volumeUnitResolution;
+    Point3i volumeUnitResolutions;
```

Comment:
Please update the code with recent commits; this is from previous code version

Reply:
done